private/buf/cmd/buf/command/beta/registry/webhook/webhookcreate/webhookcreate.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhookcreate
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/bufbuild/buf/private/buf/bufcli"
+	registryv1alpha1 "github.com/bufbuild/buf/private/gen/proto/go/buf/alpha/registry/v1alpha1"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appflag"
+	"github.com/bufbuild/buf/private/pkg/command"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	ownerFlagName        = "owner"
+	repositoryFlagName   = "repository"
+	callbackURLFlagName  = "callback_url"
+	webhookEventFlagName = "event"
+)
+
+// NewCommand returns a new Command
+func NewCommand(
+	name string,
+	builder appflag.Builder,
+) *appcmd.Command {
+	flags := newFlags()
+	return &appcmd.Command{
+		Use:   name + " <source>",
+		Short: "Create a repository webhook.",
+		Args:  cobra.ExactArgs(0),
+		Run: builder.NewRunFunc(
+			func(ctx context.Context, container appflag.Container) error {
+				return run(ctx, container, flags)
+			},
+			bufcli.NewErrorInterceptor(),
+		),
+		BindFlags: flags.Bind,
+	}
+}
+
+type flags struct {
+	WebhookEvent   string
+	OwnerName      string
+	RepositoryName string
+	CallbackURL    string
```

Comment:
I actually didn't think too hard here because I was copying from: https://github.com/bufbuild/buf/blob/main/private/buf/cmd/buf/command/beta/convert/convert.go#L60-L66

@smallsamantha do you know if there is a functional reason for having this be exported?